plugins/luarocks/llthreads2.mk

```diff
@@ -0,0 +1,23 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := llthreads2
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 0.1.3
+$(PKG)_CHECKSUM := 8c6fc7966cdcc15ae2f89f66ae72f6727a985e7d254f139ecf75a50956a3e8e4
+$(PKG)_SUBDIR   := lua-$(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $($(PKG)_SUBDIR).tar.gz
+$(PKG)_URL      := https://github.com/moteus/lua-llthreads2/archive/v$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc luarocks
+
+define $(PKG)_UPDATE
+    $(call MXE_GET_GITHUB_TAGS, moteus/lua-llthreads2) | \
+    $(SED) 's,^v,,g'
+endef
```

Comment:
Switch to new argument for `MXE_GET_GITHUB_TAGS` (also `lua-rapidjson.mk`, luasocket.mk)